mods/ra/tilesets/interior.yaml

```diff
@@ -1181,3 +1181,192 @@ Templates:
 		Categories: Floor
 		Tiles:
 			0: Clear
+	Template@291:
+		Id: 291
```

Comment:
Are these the correct template IDs for these tiles?

Try importing the "Mousetrap" aftermath mission to confirm?